datafusion/core/tests/sqllogictests/test_files/pg_compat/pg_compat_type_coercion.slt

```diff
@@ -15,108 +15,37 @@
 # specific language governing permissions and limitations
 # under the License.
 
-query ??
+query BB
 select true and true, false and false;
 ----
 true false
 
 
-onlyif DataFusion
```

Comment:
The `arrow_typeof` and `pg_typeof` checks are no longer needed for the coercion tests. Sqllogictest now checks that the columns are `Boolean`.